src/firebase.js

```diff
@@ -0,0 +1,60 @@
+import firebase from "firebase";
+import 'firebase/firestore';
+
+require("dotenv").config();
+
+// Set collection name
+const collectionName = "db_pilot_test";
```

Comment:
`db_pilot_test` is probably not the best name for our response documents - maybe `participant_responses` or have that set via env var (cc: @galenwinsor )